CHANGELOG.md

```diff
@@ -49,6 +50,12 @@
 - Initial example
 - Changelog #7
 
+## [0.1.0 UNRELEASED]
+
+## Changed
+
+- updated to fit hashistack v0.6.0
+
```

Comment:
As @pdmthorsrud says, line 53-61 should be removed. It's changelog from the template: https://github.com/fredrikhgrelland/vagrant-hashistack-template/blob/master/CHANGELOG.md